model/Licensing/Licensing.md

```diff
@@ -10,17 +10,24 @@ facilitate compliance with typical license use cases.
 ## Description
 
 The Licensing profile only contains the additional requirement that any
-Software Artifact must have a Relationship of type hasConcludedLicense.
+Software Artifact must have a `Relationship` of type `hasConcludedLicense`.
 
-Classes and Property restrictions are defined in the SimpleLicensingProfile
+Classes and Property restrictions are defined in the `SimpleLicensingProfile`
 (Classes and Properties associated with
 [license expression srings](../../annexes/SPDX-license-expressions.md)) and in
```

Comment:
Not an issue with your PR, but did notice when I was looking at this, "srings" should be "strings".

Reply:
Fixed the typo.